monkestation/code/datums/traits/neutral.dm

```diff
@@ -40,3 +40,23 @@
 		var/mob/living/carbon/human/gojira = quirk_holder
 		if(gojira.dna)
 			gojira.dna.add_mutation(GIGANTISM)
+
+/datum/quirk/fluffy_tongue
```

Comment:
Make this a negative quirk.
This is going to be a negative quirk that actually costs points.